### PR TITLE
Updated broken Dummy Rock link

### PR DESCRIPTION
**Pull request by joseph619**
Taken from discord FAQ

### CODE REVIEW

**Comment by joseph619** [comment]
How do I upload file to the website?

**Comment by Soitora** [comment]
See this PR and my commit 82230932fb8251b4612eb8be3a136b6ab5a2c7b2, where this file is.
`src/.vuepress/public/assets/misc/tachiyomi-en.dummyrock-v1.2.1-debug.apk`

**Comment by joseph619** [comment]
I will look into it later again when I'm free.

File: src/.vuepress/public/assets/misc/tachiyomi-en.dummyrock-v1.2.1-debug.apk [hunk 1]
```diff
@@ -0,0 +1 @@
+x
```
Comment (arkon) [inline src/.vuepress/public/assets/misc/tachiyomi-en.dummyrock-v1.2.1-debug.apk]: 🤔

File: src/help/faq/extensions.md [hunk 2]
```diff
@@ -200,7 +201,7 @@ View the program at [waicool20/mr2tachiyomi](https://github.com/waicool20/mr2tac
 It's not currently possible to restore backup from MangaRock officially, as the MangaRock source is dead. There is however an unofficial method to restore/backup all manga from the MangaRock source.
 
 1. Uninstall the MangaRock extension.
-1. Install the [DummyRock](https://cdn.discordapp.com/attachments/403520500443119619/663671637052489728/tachiyomi-en.dummyrock-v1.2.1-debug.apk) `.apk`. 
+1. Install the [DummyRock](https://github.com/joseph619/website/blob/master/tachiyomi-en.dummyrock-v1.2.1-debug.apk?raw=true) `.apk`.
```
Comment (arkon) [inline src/help/faq/extensions.md]: This should be pointing at the main repo in the end, not your fork.

**Pull request merged**